app/jobs/calculate_lines_of_code_job.rb

```diff
@@ -13,10 +13,8 @@ def perform(iteration)
 
     return unless iteration.submission.valid_filepaths.any?
 
-    # TODO: (Required) Set this through Exercism config
-    url = "https://internal.exercism.org/count_lines_of_code"
     body = RestClient.post(
-      url,
+      Exercism.config.lines_of_code_counter_url,
```

Comment:
See https://github.com/exercism/terraform/blob/main/terraform/dynamodb_config.tf#L353 for the production key/value pair